app/src/test/java/org/oppia/android/app/translation/AppLanguageResourceHandlerTest.kt

```diff
@@ -111,22 +111,30 @@ import javax.inject.Singleton
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(application = AppLanguageResourceHandlerTest.TestApplication::class)
 class AppLanguageResourceHandlerTest {
-  @get:Rule val initializeDefaultLocaleRule = InitializeDefaultLocaleRule()
+  @get:Rule
+  val initializeDefaultLocaleRule = InitializeDefaultLocaleRule()
```

Comment:
For this & other lines: please revert these formatting changes.

Reply:
formatting changes have now been reverted.